fury/window.py

```diff
@@ -936,6 +936,9 @@ class ReportSnapshot(object):
     if colors is not None:
         if isinstance(colors, tuple):
             colors = [colors]
+        # Drop the alpha channel if the image does not have it
+        if np.asarray(colors).shape[1] != im.shape[-1]:
```

Comment:
> I had to use asarray in case colors are passed as a list

Even if it is a list. you can do `len(colors[-1])` or `len(colors[0])` and avoid the `np.asarray`

Reply:
I think you should manage the inverse case so use '>' operator. 

Can you add some check for the input image shape

Reply:
> Even if it is a list. you can do `len(colors[-1])` or `len(colors[0])` and avoid the `np.asarray`

Done in c1ca1e7.

> Can you add some check for the input image shape

Done in c1ca1e7.